testApi/src/main/scala/africa/absa/testing/scapi/rest/response/LogResponseAction.scala

```diff
@@ -54,10 +57,13 @@ object LogResponseAction extends ResponsePerformer {
    * @throws IllegalArgumentException if the response action's name is not recognized.
    */
   def performResponseAction(response: Response, responseAction: ResponseAction): Boolean = {
```

Comment:
The returned boolean value is consumed in logic of SuiteRunner where this information is used for SuiteResults purposes.  This could be replaced be throwing an exception when logic failed.

Reply:
So my idea would be:
- Remove all the `true` from the small `logSomething` methods. 
- Replace it here with something better, like that `SuiteResult.Success`
- Replace that throw with `SuiteResult.Failure`

How does that sound?

Reply:
Fixed in commit c203ab6e0a2670215c2b1f00296202274cfbd634.